pkg/secretdump/secretdump_test.go

```diff
@@ -0,0 +1,84 @@
+package secretdump_test
```

Comment:
You can't have two packages under the same directory.
Our guide lines are to use the same package and directory for the code and the tests

Reply:
Will fix. This was generated by `ginkgo generate secretdump` so I figured it was alright

Reply:
Just some question - because you put the tests under the same package, and tests begin with `Test...`, since `Test` starts with an uppercase letter, doesn't that mean that all packages export their tests under their namespace? That's a bit weird - test functions would appear in IDE autocompletion

Reply:
Are you sure about auto completion? i think that goland ignore tests, not sure about vscode 
The benefit is that you can call internal components and variables inside the tests, you will not be able to do the same in a different package

Reply:
> Are you sure about auto completion? i think that goland ignore tests, not sure about vscode

Not at all, that's just that what I assumed/understood would happen. If they have special handling for that then I guess there's no problem

> The benefit is that you can call internal components and variables inside the tests, you will not be able to do the same in a different package

I see, makes sense. Thanks for the explanation